app/utils/fetch-selection-summary.js

```diff
@@ -0,0 +1,16 @@
+import Environment from '../config/environment';
+
+const { SupportServiceHost } = Environment;
+
+const SELECTION_SUMMARY_API_URL = (survey = 'decennial', summaryVars = 'pop1,hunits', geoid = '0') => `${SupportServiceHost}/summary/${survey}/${summaryVars}/${geoid}`;
+
+
+export default async function fetchSelectionSummary(survey, summaryVars, geoid) {
```

Comment:
@godfreyyeung do you think this should me in a new Model? My understanding was that any async calls in Ember should be encapsulated in a Model but I'm not necessarily opposed to forgoing that for this functionality.

Reply:
in this case it's a lot of overhead and work to rework it into a model. It would require the API to deliver JSON-API and we'd have to rework the selection service, and all dependents of the selection service